package/ios/AppleLogger.mm

```diff
@@ -8,9 +8,11 @@
 #import "MmkvLogger.h"
 #import <Foundation/Foundation.h>
 
-void MmkvLogger::log(const std::string& tag, const std::string& message) {
+void MmkvLogger::log(const std::string& tag, const std::string& formatString, Args... args) {
```

Comment:
same here, not a template.

we can't even template this because our implementation is in two differnt spots (ios and android). so we might not be able to support `Args...`.